libkbfs/tlf_journal.go

```diff
@@ -813,6 +826,56 @@ func (e errTLFJournalNotEmpty) Error() string {
 	return "tlfJournal is not empty"
 }
 
+func (j *tlfJournal) checkServerForConflicts(ctx context.Context) error {
```

Comment:
This is locked by `flushLock`, not sure how you want to capture that in the name (important because it reads/writes `lastServerMDCheck`).